tests/bot_tests/SOA_test.py

```diff
@@ -26,11 +26,11 @@
 
 class TestSOABot():
     def test(self):
-        # start_io_loop(self.test_play)
+        start_io_loop(self.test_play)
         # start_io_loop(self.test_stance)
```

Comment:
remove commented out code

Reply:
we can run only one test at a time due to ioloop. So those test functions were there even with comments, just to later uncomment and run test one by one.

Reply:
There is certainly a way around that. What have you tried/had difficulty with?

Reply:
I have tried `startioloop(test)` but it doesn't work. but this is not a focus for this branch. The `SOA_test.py` file is already in the main right now.